CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.58.0] - 2021-04-19
+
+### Changed
+
+- Upgrade play-frontend-govuk
+
+### Compatible with
+
+- [hmrc/hmrc-frontend v1.29.0](https://github.com/hmrc/hmrc-frontend/releases/tag/v1.29.0)
+- [hmrc/play-frontend-govuk v0.71.0](https://github.com/hmrc/play-frontend-govuk/releases/tag/v0.71.0)
```

Comment:
@matthewmascord just checking, should this be the same as the version in project/LibDepedencies?

Reply:
Good catch 👍 🤦